webui/src/pages/namespace-detail/namespace-detail.tsx

```diff
@@ -0,0 +1,249 @@
+            </>;
+        }
+
+        const classes = this.props.classes;
+        return <>
+            <DelayedLoadIndicator loading={this.state.loading} />
+            <Box className={classes.head}>
+                <Container maxWidth='xl'>
+                    <Box className={classes.header}>
+                        <Grid container>
+                            <Grid item>
+                                <img src={namespaceDetails.logo || this.context.pageSettings.urls.extensionDefaultIcon}
+                                    className={`${classes.namespaceLogo} ${classes.badgePadding}`}
+                                    alt={namespaceDetails.displayName || namespaceDetails.name} />
+                            </Grid>
+                            <Grid item xs={7}>
+                                <Grid container spacing={2}>
+                                    <Grid item>
+                                        <Typography variant='h5'>{namespaceDetails.displayName || namespaceDetails.name}</Typography>
+                                    </Grid>
+                                </Grid>
+                                <Grid container spacing={2}>
+                                    <Grid item className={classes.descriptionPadding}>
+                                        {
+                                            namespaceDetails.description
+                                            ? truncateReadMore
+                                              ? <Truncate lines={2} ellipsis={<span>... <Link color='secondary' onClick={this.readMore}>Read more</Link></span>}>
+                                                { namespaceDetails.description }
+                                              </Truncate>
+                                              : <Typography>{ namespaceDetails.description }</Typography>
+                                            : null
+                                        }
+                                    </Grid>
+                                </Grid>
+                                <Grid container spacing={2}>
+                                    {
+                                        namespaceDetails.companyLink
+                                        ? <Grid item><Link color='secondary' href={namespaceDetails.companyLink}>{namespaceDetails.companyLink}</Link></Grid>
+                                        : null
+                                    }
+                                    {
+                                        namespaceDetails.companyLink && namespaceDetails.supportLink ? <Divider className={classes.linksDivider} orientation='vertical' flexItem /> : null
+                                    }
+                                    {
+                                        namespaceDetails.supportLink
+                                        ? <Grid item><Link color='secondary' href={namespaceDetails.supportLink}>{namespaceDetails.supportLink}</Link></Grid>
```

Comment:
see the comment right above

```suggestion
                                        ? <Grid item><Link color='secondary' href={namespaceDetails.supportLink}>{namespaceDetails.supportLink.replace(/https?:\/\//, '')}</Link></Grid>
```